### PR TITLE
Created function chapter in doc

### PR DESCRIPTION
**Pull request by 0ctagon**
Right now, the ability to plot functions is lost two times in the middle of the ``1d histogram`` chapter. This new chapter gather everything that is linked to function plotting.

See doc here: https://plothist.readthedocs.io/en/function_doc_category/basics/1d_func.html

### CODE REVIEW

**Comment by 0ctagon** [comment]
Will correctly build after #159 is merged

File: docs/basics/1d_func.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+.. code-block:: python
+
+    from plothist import get_dummy_data
+
+    df = get_dummy_data()
+
+.. note::
+   This page presents functions of ``plothist`` step by step and gives information about how to use them.
+   To reproduce the examples, please visit the `plot gallery <https://plothist.readthedocs.io/en/latest/example_gallery/>`_, because it contains a standalone script for each example, that you can run directly.
```
Comment (cyrraz) [inline docs/basics/1d_func.rst]: example gallery

File: docs/basics/1d_func.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+.. note::
+    All the functions used in the examples below can take a lot more arguments to customize the plot, see the :ref:`documentation-label` for more details.
+
+
+Simple function
+===============
+
+Everything presented for the histogram is also true to plot functions using the :func:`plot_function() <plothist.plotters.plot_function>` function:
```
Comment (cyrraz) [inline docs/basics/1d_func.rst:26]: "Everything presented for the histogram is also true to plot functions"

> Plothist supports plotting of functions

**Review by cyrraz — APPROVED** [review]
Thank you, just one comment

**Pull request merged**